app/views/candidates/registrations/confirmation_emails/show.html.erb

```diff
@@ -23,6 +23,6 @@
       If you can’t find or haven’t received the email use the following link to
       resend it:
     </p>
-    <%= button_to 'Resend link', @resend_path, class: 'govuk-button govuk-button--secondary' %>
+    <%= button_to 'Resend link', @resend_link, class: 'govuk-button govuk-button--secondary' %>
```

Comment:
`@resend_link` is nil if we try and resend the confirmation email, causing button_to to point at `/candidates/schools/138742/registrations/confirmation_email` rather than `/candidates/schools/138742/registrations/resend_confirmation_email` like on master.
Can probably just rename `@resend_path` in `ConfirmationEmailsController#show` and it should work ok

Reply:
Good catch - that was me renaming the variable without sufficient checking

Reply:
I've created a separate story to track this - SE-1257

Theres a few places which will need feature flagging before the epic can get merged to master but I want to discuss how this roles out first anyway

Reply:
Change pushed to update the controller and spec